### ISSUE TITLE
[Org Sign Up -  Identity Verification] No way to fill out form using only the keyboard

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5880]
## Area or Component

Org sign up flow - Experian Identity Verification section

## Quick Summary
There is no way to keyboard "tab" through the different radio button groups.

## Accessibility Violation

[Deque Link](https://docs.deque.com/issue-help/1.0.0/en/keyboard-inaccessible)

## Action Requested

Ensure the form can be navigable by keyboard according to this: https://www.w3.org/WAI/ARIA/apg/patterns/radio/

## Screenshots (optional)

<img width="595" alt="Screenshot 2023-05-26 at 13 24 12" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/aa2fbd3d-1db8-4489-93f5-72f20376271f">

## Source Code (optional)

[which file or line is the issue attached to?]: #
